_locales/es/messages.json

```diff
@@ -1044,12 +1050,18 @@
     "receiverRcInterpolationInterval": {
         "message": "Intervalo de Interpolaci\u00f3n RC [ms]"
     },
+    "pidTuningDtermSetpointTransition": {
+        "message": "Punto de Ajuste de Transici\u00f3n de D"
+    },
     "pidTuningPtermSetpoint": {
```

Comment:
This should be removed, it's not used any more, and was removed from the English file.